templates/scale_up_resources.j2

```diff
@@ -273,6 +273,6 @@ primitive rsc_exporter_{{ sid }}_HDB{{ instance }} systemd:prometheus-hanadb_exp
     meta resource-stickiness=0 \
     meta target-role=Started
 
-colocation col_exporter_{{ sid }}_HDB{{ instance }} 1: rsc_exporter_{{ sid }}_HDB{{ instance }}:Started msl_SAPHana_{{ sid }}_HDB{{ instance }}:Master
+colocation col_exporter_{{ sid }}_HDB{{ instance }} -inf: rsc_exporter_{{ sid }}_HDB{{ instance }}:Started msl_SAPHana_{{ sid }}_HDB{{ instance }}:Slave
```

Comment:
I think we didn't understand each other. What I suggest is the next:

```
primitive rsc_exporter_{{ sid }}_HDB{{ instance }} systemd:prometheus-hanadb_exporter@{{ sid }}_HDB{{ instance }} \
    op start interval=0 timeout=100 \
    op stop interval=0 timeout=100 \
    op monitor interval=10 \
    meta resource-stickiness=0 \
    meta target-role=Started

colocation col_exporter_{{ sid }}_HDB{{ instance }} 1: rsc_exporter_{{ sid }}_HDB{{ instance }}:Started msl_SAPHana_{{ sid }}_HDB{{ instance }}:Master

colocation col_exporter_{{ sid }}_HDB{{ instance }}_Slave -inf: rsc_exporter_{{ sid }}_HDB{{ instance }}:Started msl_SAPHana_{{ sid }}_HDB{{ instance }}:Slave

```

Basically, I proposed to add an additional `colocation` to the ones you already added keeping the `resource-stickines=0` value too.
At least this is what I tested and worked.

Reply:
If you tried only with the `Slave` colocation and it works it is fine for anyway.

Reply:
I got your point. But this would add more complexity to the cluster and dependencies between HANA master and slave. I've tested the proposed change and have added only the colocation constraint for the slave and it works.

Reply:
Ok